workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/NodeCollection.java

```diff
@@ -445,27 +441,17 @@ public String toString() {
 
     public String toXML(Set<String> printedNodeNames) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "toXML" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)